hadoop-ozone/dist/src/main/smoketest/s3/bucketdelete.robot

```diff
@@ -43,3 +43,21 @@ Delete non-existent bucket
     ${randStr} =   Generate Ozone String
     ${result} =    Execute AWSS3APICli and checkrc    delete-bucket --bucket nosuchbucket-${randStr}    255
                    Should contain                     ${result}                              NoSuchBucket
+
+Delete bucket with incomplete multipart uploads
+    ${bucket} =                Create bucket to be deleted
+
+    # initiate incomplete multipart uploads (multipart upload is initiated but not completed/aborted)
+    ${initiate_result} =       Execute AWSS3APICli     create-multipart-upload --bucket ${bucket} --key incomplete-multipartkey
+    ${uploadID} =              Execute and checkrc     echo '${initiate_result}' | jq -r '.UploadId'    0
```

Comment:
Nit: `Execute and checkrc` with 0 expected exit code can be simplified to `Execute`.

https://github.com/apache/ozone/blob/ff3c071f52310289c01dafbe0a6939e8e65c5398/hadoop-ozone/dist/src/main/smoketest/lib/os.robot#L19-L22

```suggestion
    ${uploadID} =              Execute                 echo '${initiate_result}' | jq -r '.UploadId'
```

Reply:
Updated accordingly.